content/en/opentelemetry/guide: mention Datadog Processor (#16439)
DataDog/documentation: content/en/opentelemetry/guide/ingestion_sampling_with_opentelemetry.md

**Note**: Tail sampling's main limitation is that all spans for a given trace must be received by the same collector instance for effective sampling decisions. If the trace is distributed across multiple collector instances, there’s a risk that some parts of a trace are dropped whereas some other parts of the same trace are sent to Datadog.

> With collector-level tail-based sampling, the APM metrics are computed on the sampled traffic that the Datadog Exporter sees. For this reason, please use the [`datadogprocessor`][11] to ensure that APM metrics are computed for 100% of your traffic. The Datadog Processor is available starting v0.69.0.

Reviewer: ```suggestion
To ensure that APM metrics are computed based on 100% of the applications' traffic while using collector-level tail-based sampling, preprend the [Datadog Metrics Processor][11] `datadogprocessor`. The processor is available from the v0.69.0 of the OpenTelemetry collector (contrib).
```

Author: When we say "prepend the processor" it has to be followed by what it should be prepended to. So maybe saying "prepend the `datadogprocessor` to the sampler in your traces pipeline". Otherwise let's leave it as it was. Also, let's not call it "Datadog *Metrics* Processor". It is called "Datadog Processor" everywhere. So, in the end, I'm suggesting:

> To ensure that APM metrics are computed based on 100% of the applications' traffic while using collector-level tail-based sampling, prepend the [Datadog Processor][11] before any sampling in your traces pipeline. The processor is available starting from v0.69.0 in the OpenTelemetry Collector contrib release.

What do you think?

Reviewer: i modified the wording, lmk what you think

Author: It is much better but I would not say "prepend the Datadog Processor datadogprocessor"... Having it twice is too much, repetitive and not very relevant because it is used as "datadog". "datadogprocessor" is just the name of the folder. Let me commit the changes and will ask you for another review.

Author: I applied your suggestion with minor grammar corrections. Let me know what you think.

> To ensure that APM metrics are computed based on 100% of the applications' traffic while using collector-level tail-based sampling, preprend the [Datadog Processor][11] in front of your sampling processor in the collectors' traces pipeline. The processor is available starting from the v0.69.0 of the OpenTelemetry Collector Contrib release.

Reviewer 2: ```suggestion
To ensure that APM metrics are computed based on 100% of the applications' traffic while using collector-level tail-based sampling, preprend the [Datadog Processor][11] in front of your sampling processor in the collectors' traces pipeline. The processor is available with OpenTelemetry Collector Contrib v0.69.0+.
```

Author: Thanks! I think we were both having trouble with this one :))

> Use the [ingestion volume control guide][8] to read more about the implications of setting up trace sampling on trace analytics monitors and metrics from spans.

Reviewer 2: ```suggestion
See the [ingestion volume control guide][8] for information about the implications of setting up trace sampling on trace analytics monitors and metrics from spans.
```
